docs/jwk2pem.py

```diff
@@ -6,14 +6,29 @@
 # details.
 #
 """A helper code to convert JWK (retrieved from oidc) to PEM format."""
-import jwcrypto.jwk
 
-key_dict = {
-    "e": "AQAB",
-    "kty": "RSA",
-    "n": "mho5h_lz6USUUazQaVT3PHloIk_Ljs2vZl_RAaitkXDx6aqpl1kGpS44eYJOaer4oWc6_QNaMtynvlSlnkuWrG765adNKT9sgAWSrPb81xkojsQabrSNv4nIOWUQi0Tjh0WxXQmbV-bMxkVaElhdHNFzUfHv-XqI8Hkc82mIGtyeMQn-VAuZbYkVXnjyCwwa9RmPOSH-O4N4epDXKk1VK9dUxf_rEYbjMNZGDva30do0mrBkU8W3O1mDVJSSgHn4ejKdGNYMm0JKPAgCWyPWJDoL092ctPCFlUMBBZ_OP3omvgnw0GaWZXxqSqaSvxFJkqCHqLMwpxmWTTAgEvAbnw",
-}
+import sys
 
-key = jwcrypto.jwk.JWK(**key_dict)
-pem = key.export_to_pem(False, False)
-print(pem)
+
+def export_key() -> None:
+    """Export the key from JWK to PEM format."""
+    try:
+        import jwcrypto.jwk  # noqa
+    except ImportError:
+        print("Please install jwcrypto: pip install jwcrypto")
+        sys.exit(1)
+
+    # this key was downloaded from perun
+    key_dict = {
+        "e": "AQAB",
+        "kty": "RSA",
+        "n": "mho5h_lz6USUUazQaVT3PHloIk_Ljs2vZl_RAaitkXDx6aqpl1kGpS44eYJOaer4oWc6_QNaMtynvlSlnkuWrG765adNKT9sgAWSrPb81xkojsQabrSNv4nIOWUQi0Tjh0WxXQmbV-bMxkVaElhdHNFzUfHv-XqI8Hkc82mIGtyeMQn-VAuZbYkVXnjyCwwa9RmPOSH-O4N4epDXKk1VK9dUxf_rEYbjMNZGDva30do0mrBkU8W3O1mDVJSSgHn4ejKdGNYMm0JKPAgCWyPWJDoL092ctPCFlUMBBZ_OP3omvgnw0GaWZXxqSqaSvxFJkqCHqLMwpxmWTTAgEvAbnw",
```

Comment:
Maybe rather take this from ENV? WHat if the key later changes - would prompt for new release